src/app/jobs/pages/jobs/jobs.component.ts

```diff
@@ -160,4 +171,15 @@ export class JobsComponent implements OnInit, AfterViewInit {
     const filterValue = (event.target as HTMLInputElement).value;
     this.dataSource.filter = filterValue.trim().toLowerCase();
   }
+
+  openDialog() {
+    const dialogRef = this.dialog.open(JobAddDialogComponent, {
+      width: "550px",
+      height: "500px",
+    });
+
+    dialogRef.afterClosed().subscribe(result => {
```

Comment:
To fix the ESLint warning:

```suggestion
    dialogRef.afterClosed().subscribe(() => {
```

Or even better: don't subscribe to the event. The `console.log` call is totally not needed.